Create prathamesh.json

prathamesh-2002 opened the pull request:
<!--
!!!
YOU MUST FILL OUT THIS TEMPLATE FOR YOUR PR TO BE ACCEPTED.
IF YOU DO NOT FILL OUT THIS PR TEMPLATE TO ITS ENTIRETY, YOUR PR WILL BE IMMEDIATELY DENIED.
!!!
-->

# Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain **MUST** pass **ALL** the indicated requirements below.

<!-- Change each checkbox to [x] to mark it as checked. Do not keep the spaces between the parentheses. -->
- [x] I have **read** and **understood** the [Terms of Service](https://is-a.dev/terms). <!-- Your domain MUST follow the TOS to be approved. -->
- [x] My domain is **not** for commercial use. <!-- e.g. Your domain should not be selling products & services -->
- [x] My file is in the `domains` directory and has the `.json` file extension.
- [x] My file's name is lowercased and alphanumeric. <!-- Your file's name is yourname.json, not YourName.json or your_name.json. -->
- [x] My website is **reachable**.
- [x] I have **completed** my website. <!-- We do not permit simple "Hello, world!" websites. -->
- [x] My website is related to **software development**. <!-- We don't permit websites for domains unrelated to development (e.g. Discord servers, gaming guilds, etc.) -->
- [x] I have provided sufficient contact information in the `owner` key. <!-- Provide your email in the `email` field or another platform (e.g., X, Discord) for contact. -->
- [x] I have followed and read the [documentation](https://docs.is-a.dev).
- [x] I understand my domain will be removed if I violate the [Terms of Service](https://is-a.dev/terms).

# Website Preview
<!-- Provide a link or preview of your website below. If you can't make the website visible, then a screenshot of the website also works. -->
https://prathamesh-2002.github.io/portfolio/

prathamesh-2002 committed:
Create prathamesh.json

prathamesh-2002 committed:
Merge branch 'main' into main

justdevw reviewed (commented):
uh you cant just claim subdomain that already owned by someone

prathamesh-2002 commented:
Hey @JustDeveloper1 , I did see an automated domain cleanup for this domain and the domain still isn't active. That's why I opened a pr. Incase this domain isn't eligible, should I edit this pr with a new domain name.

![#16692](https://github.com/user-attachments/assets/2ac3a262-04ed-453d-b4c9-f5f0d81637e1)

JustDeveloper1 commented:
> Hey @JustDeveloper1 , I did see an automated domain cleanup for this domain and the domain still isn't active. That's why I opened a pr. Incase this domain isn't eligible, should I edit this pr with a new domain name.
> 
> ![#16692](https://private-user-images.githubusercontent.com/54547411/412551721-2ac3a262-04ed-453d-b4c9-f5f0d81637e1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzkzODM0NzAsIm5iZiI6MTczOTM4MzE3MCwicGF0aCI6Ii81NDU0NzQxMS80MTI1NTE3MjEtMmFjM2EyNjItMDRlZC00NTNkLWI0YzktZjVmMGQ4MTYzN2UxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAyMTIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMjEyVDE3NTkzMFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWYzZjBlNWZiMzViZjM4ZTAyNTdjY2Y3N2JjM2JlMzA5ZTlmYjM0OThlMTM5ZjgxMmY4OGZmYmY2N2EyZjZhYWQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.VzFu-nGwx4VVk_kIL3Uh2plJ3aNKnwFvpGrYHSGDIrg)

that domain cleanup was cancelled, so you need to rename your subdomain you want to get

JustDeveloper1 commented:
if the website at prathamesh.is-a.dev is inaccessible or violate is-a.dev you can report it [here](https://github.com/is-a-dev/register/issues/new?labels=report-abuse&template=report-abuse.md&title=Report+abuse)

prathamesh-2002 commented:
Okay, I'll do it rn. Thanks for the prompt response.

prathamesh-2002 committed:
Update and rename prathamesh.json to prxthamesh.json

Added a new domain name. 

Meanwhile I'll also raise a report for the previously requested domain inactivity.

prathamesh-2002 commented:
@JustDeveloper1 please check the updated pr

JustDeveloper1 commented:
> @JustDeveloper1 please check the updated pr

you need to restore that domain that not owned by you back
![image](https://github.com/user-attachments/assets/ba512cc3-5b06-4b43-a98d-8a7998ac2495)

JustDeveloper1 commented:
also i am not a maintainer so i cant merge or close or request changes on your pull request

prathamesh-2002 committed:
Merge branch 'main' into main

prathamesh-2002 commented:
> also i am not a maintainer so i cant merge or close or request changes on your pull request

Ah, I see. I'll close this pr as it has gotten messy and just raise a new one.
Thanks for helping out.
